application/services/responses/store_responses.rb

```diff
@@ -43,7 +44,7 @@ def create_response_entities_arr(input)
         input[:pages].each do |page|
           page_index = page.index
           page.items.each do |item|
-            next if item.type == 'Description' || item.type == 'Section Title' || item.type == 'Divider' || item.type == 'Jump to page'
```

Comment:
One last suggestion!  What is the reason we are skipping these item types (Description, Section Title, Divider, Jump to page, Disable previous page)? What makes them special?  If you can figure it out, please rename `ITEM_TYPE` constant to that name (e.g., `NONRESPONSE_ITEMS` – I'm guessing because the method is called `create_response_entities_arr`).